mlflow/cli.py

```diff
@@ -71,8 +71,12 @@ def cli():
               help="Only valid when `mode` is local."
                    "MLflow will download artifacts from distributed URIs passed to parameters of "
                    "type 'path' to subdirectories of storage_dir.")
+@click.option("--run-id", metavar="RUN_ID",
+              help="Note: this argument is used internally by the MLflow project APIs "
```

Comment:
Maybe flip these, first say what it does and then say that this is mainly for internal use?